wrappers/rust/icicle-core/src/ntt/tests.rs

```diff
@@ -331,6 +331,9 @@ where
                         config.ordering = ordering;
                         let mut batch_ntt_result = vec![F::zero(); batch_size * test_size];
                         for alg in [NttAlgorithm::Radix2, NttAlgorithm::MixedRadix] {
+                            if alg == NttAlgorithm::Radix2 && ordering as u32 > 3 {
+                                continue;
```

Comment:
Add a comment on why we are skipping these runs

Reply:
added comment